CoDEC_Explorer/app.R

```diff
@@ -491,13 +557,16 @@ server <- function(input, output, session) {
         mutate(bi_class = paste0(as.numeric(bi_x), "-", as.numeric(bi_y)))
       
       out <- out |> 
-        mutate(out_lab = paste(xvar(), ": ", get(xvar()), "<br>",
-                               yvar(), ": ", get(yvar())))
+        mutate(out_lab = paste(xvar(), ": ", round(get(xvar()),2), "<br>",
+                               yvar(), ": ", round(get(yvar()),2)))
       
       pal <- colorFactor(codec_bi_pal, factor(out$bi_class, levels = c("1-1","2-1","3-1",
                                                                        "1-2","2-2","3-2",
                                                                        "1-3","2-3","3-3")))
       
+      out <- sf::st_transform(out, crs = "NAD83")
```

Comment:
For some reason, this was the only way I could get the map to draw. I tried setting them to 5072 or using that method to set it to the crs of the upstream dataframe, but the map doesn't draw and I get an additional warning:
`Warning: sf layer is not long-lat data
Warning: sf layer has inconsistent datum (+proj=aea +lat_0=23 +lon_0=-96 +lat_1=29.5 +lat_2=45.5 +x_0=0 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs).
Need '+proj=longlat +datum=WGS84'`

And this appears even though the appropriate projection is applied to the mapping dataframe. After switching from  tigris for the shapes, I had to add the calls to st_transform(crs = "NAD83") for the maps to plot. It may be a leaflet thing as mapview testing didn't give me these warnings

Reply:
Leaflet draws maps using lat/lon coordinates, not projected coordinates, so you won't be able to use 5072 or another projection to do that.  I think mapview does this under the hood for you, but maybe not leaflet.

Reply:
Ahh, looks like that was it! At the top of the "stream" I transformed to 4326, eliminating all of the warnings and fixing the map-click highlight. `interpolate` transforms to 5072, but I can transform back to 4326. Does that make sense to set it all to 4326?

Reply:
awesome.  yes, that sounds like a good idea since we living in leaflet here and not worried about using any specific projection-based calculations